pandas/core/computation/eval.py

```diff
@@ -156,9 +160,17 @@ def _check_for_locals(expr, stack_level, parser):
                 raise SyntaxError(msg)
 
 
-def eval(expr, parser='pandas', engine=None, truediv=True,
-         local_dict=None, global_dict=None, resolvers=(), level=0,
-         target=None, inplace=False):
+def eval(expr: str,
+         parser: str = 'pandas',
+         engine: str = None,
+         truediv: bool = True,
+         local_dict: Dict[str, Any] = None,
+         global_dict: Dict[str, Any] = None,
+         resolvers: Iterable = (),
```

Comment:
I think this needs to be `List[Dict]` per documentation (may also help CI failure)